docs/pod/zfsbootmenu.7.pod

```diff
@@ -18,9 +18,25 @@ These options are set on the kernel command line when booting the initramfs or U
 
 When creating an initramfs or UEFI bundle, the I</etc/hostid> from the system is copied into the target. If this image will be used on another system with a different hostid, replace B<E<lt>hostidE<gt>> with the desired hostid, as an eight-digit hexadecimal number, to override the value contained within the image.
 
+=item B<zbm.prefer>
+
+ZFSBootMenu will attempt to import as many pools as possible to identify boot environments and will, by default, look for the I<bootfs> property on the first imported pool (sorted alphabetically) to select the default boot environment. This option controls this behavior.
+
+=over 2
+
 =item B<zbm.prefer=E<lt>poolE<gt>>
 
-ZFSBootMenu will attempt to import as many pools as possible to identify boot environments and will, by default, look for the I<bootfs> property on the first imported pool (sorted alphabetically) to select the default boot environment. If you have multiple pools, replace B<E<lt>poolE<gt>> with the name of your preferred pool to override the default. If B<E<lt>poolE<gt>> is the name of a pool followed by a literal I<!> character, ZFSBootMenu will insist on successfully importing the named pool before attempting to import any others.
+The simplest form attempts to import B<E<lt>poolE<gt>> before any other pool. The I<bootfs> value from this pool will control the default boot environment.
+
+=item B<zbm.prefer=E<lt>poolE<gt>I<!>>
+
+If a literal I<!> has been appended to the pool name, ZFSBootMenu will insist on successfully importing the named pool before attempting to import any others
```

Comment:
Missing period at the end of this sentence

Reply:
fixed